visualizer.py

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/python3
+"""
+Author: Vertigo Designs, Ryan Summers
+
+Description: Bokeh application for serving Stabilizer stream visuals.
+"""
+import argparse
+import asyncio
+import enum
+import logging
+import socket
+from typing import List, Mapping
+
+import tornado
+
+import bokeh.plotting
+import bokeh.layouts
+import bokeh.document
+import bokeh.palettes
+import bokeh.io
+import bokeh.models
+import bokeh.server.server
+import numpy as np
+
+from miniconf import Miniconf
+from stabilizer.stream import StabilizerStream, Trace
+
+# The sample rate of stabilizer.
+SAMPLE_RATE_HZ = 100e6 / 128
```

Comment:
If you push the `to_traces()` down into stabilizer, then this should also go there. It's the time scale.

Reply:
Agreed on this front. One thing I'm curious to hear your thoughts on is managing the sample rate.

Technically, the user can modify Stabilizer apps to use different sampling rates with relative ease. However, that's a compile time constant and we don't currently report it in the stream data. Perhaps we could report the effective sample rate as a config parameter over out-of-band telemetry? Or perhaps add it to the header?